test/test_response.py

```diff
@@ -332,6 +332,25 @@ def test_decode_zstd(self) -> None:
         r = HTTPResponse(fp, headers={"content-encoding": "zstd"})
         assert r.data == b"foo"
 
+    @onlyZstd()
+    def test_decode_multiframe_zstd(self) -> None:
+        data = (
+            # Zstandard frame
+            zstd.compress(b"foo")
+            # skippable frame (must be ignored)
+            + bytes.fromhex(
+                "50 2A 4D 18"  # Magic_Number (little-endian)
+                "07 00 00 00"  # Frame_Size (little-endian)
+                "00 00 00 00 00 00 00 "  # User_Data
```

Comment:
nit: Can you please remove the trailing space after the last 00?

Reply:
Sorry for that, fixed!